news/2024/2024-08-14-osu!-deskmats.md

```diff
@@ -0,0 +1,65 @@
+---
+layout: post
+title: osu! deskmats
+date: 2024-08-14 15:00:00 +0000
+---
+
+You sit at your desk every day. You play osu! every day. You can see where we're going with this.
+
+![](/wiki/shared/news/2024-08-14-osu-deskmats/banner.jpg)
+
+A couple years ago, we celebrated osu!'s 15th anniversary with limited edition deskmats, and some of you pointed out that these "deskmats" were barely big enough to fit a keyboard.
+
+![](/wiki/shared/news/2024-08-14-osu-deskmats/twitter.png)
+
+So we expanded.
+
+![](/wiki/shared/news/2024-08-14-osu-deskmats/rollout.gif)
+
+## circle clicker deskmat
```

Comment:
the casing sure is weirdly inconsistent. it seems these are product names so i'd either prepend "The" or break the rules and make "fanart" lowercase in the next heading

Reply:
fanart isn't a title in the next section though

Reply:
i'm not sure what you mean

`## Fanart deskmats` -> `## fanart deskmats` was what i was suggesting (would make all headings lowercase)

but well if not then the other solution works just as well :)

Reply:
i mean "circle clicker"/"sakura pippi"/"everything" are used as titles, so it makes sense to keep them stylized with lowercase. "Fanart deskmats" is used as a descriptive header, so i think capitalizing under normal rules makes sense, and making it lowercase wouldn't

Reply:
new links will hopefully make this understandable